src/testRunner/unittests/semver.ts

```diff
@@ -167,6 +168,7 @@ namespace ts {
                 ["<=2", "2.0.0"],
                 ["<=2.1", "2.1.0"],
                 ["<=2.0.1", "2.0.1"],
+                ["<=4.8", "4.9.0-beta", false],

```

Comment:
```suggestion
                ["<=4.8", "4.9.0-beta", false],
                [">=4.9", "4.9.0-beta", true],
```

Reply:
`>=4.9` [matches `4.9.0-beta`](https://jubianchi.github.io/semver-check/#/%3E%3D4.9.0/4.9.0-beta), but `>4.9` [does not](https://jubianchi.github.io/semver-check/#/%3E4.9/4.9.0-beta).

semver is weird